tests/catch/CMakeLists.txt

```diff
@@ -46,32 +50,33 @@ message(STATUS "ROCM Path: ${ROCM_PATH}")
 
 
 if(UNIX)
-    set(CMAKE_CXX_COMPILER "${HIP_PATH}/bin/hipcc")
-    set(CMAKE_C_COMPILER "${HIP_PATH}/bin/hipcc")
-    set(HIPCONFIG_EXECUTABLE "${HIP_PATH}/bin/hipconfig")
-    execute_process(COMMAND perl ${HIPCONFIG_EXECUTABLE} --version
-                    OUTPUT_VARIABLE HIP_VERSION
-                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+    set(CMAKE_CXX_COMPILER "${HIP_PATH}/bin/hipcc.bin")
+    set(CMAKE_C_COMPILER "${HIP_PATH}/bin/hipcc.bin")
+    set(HIPCONFIG_EXECUTABLE "${HIP_PATH}/bin/hipconfig.bin")
+    # execute_process(COMMAND ${HIPCONFIG_EXECUTABLE} --version
+    #                 OUTPUT_VARIABLE HIP_VERSION
+    #                 OUTPUT_STRIP_TRAILING_WHITESPACE)
+    # message("hipcc: HIP_VERSION ${HIP_VERSION}")
 else()
-    # using cmake_path as it handles path correctly.
-    # Set both compilers else windows cmake complains of mismatch
-    cmake_path(SET CMAKE_CXX_COMPILER "${HIP_PATH}/bin/hipcc.bat")
-    cmake_path(SET CMAKE_C_COMPILER "${HIP_PATH}/bin/hipcc.bat")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --rocm-path=${ROCM_PATH}")
-    set(HIPCONFIG_EXECUTABLE "${HIP_PATH}/bin/hipconfig.bat")
-    execute_process(COMMAND ${HIPCONFIG_EXECUTABLE} --version
-                    OUTPUT_VARIABLE HIP_VERSION
-                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+    # # using cmake_path as it handles path correctly.
```

Comment:
On Windows, hipcc.bat is being used to invoke hipcc perl wrapper. Commenting of these lines is not correct.